crates/pallet-balanced-currency-swap-bridges-initializer/src/mock/mod.rs

```diff
@@ -0,0 +1,90 @@
+//! The mocks for the pallet testing.
+
+use frame_support::{parameter_types, sp_io, sp_runtime::BuildStorage};
+
+use crate::{self as pallet_bridges_initializer_currency_swap};
```

Comment:
This should be adjusted too

Reply:
My bad, thanks - https://github.com/humanode-network/humanode/pull/716/commits/a2263507d50bf848858c70768f4432e2d24fe3ec.